### PR TITLE
Add Code of Conduct guidelines

### PR DESCRIPTION
**Pull request by ylluminate**
I think it makes sense to have a sensible and concise code of conduct (same as Ruby's) that will encourage general good will towards each other in the community as V moves forward.

### CODE REVIEW

**Comment by ylluminarious** [comment]
I second this as a good choice for a code of conduct. Simple, short, and sweet.

File: CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+This document provides community guidelines for a safe, respectful, productive,
+and collaborative place for any person who is willing to contribute to the Opal
```
Comment (ylluminarious) [inline CONDUCT.md]: @ylluminate You probably want to change the word "Opal" here to "V programming language" or something like that.

**Comment by medvednikov** [comment]
I like simplicity, so I think for now "be nice and respectful" is enough :)

**Comment by Oxylibrium** [comment]
+1 on @medvednikov's opinion, I personally just use `Be excellent to each other.` as a standard behaviour guideline.

File: CONDUCT.md (hunk 1 above)
Comment (ylluminate) [reply CONDUCT.md]: Sorry, you're right. I just grabbed it real quick while I was thinking about it without modification, but yeah.

**Comment by ylluminate** [comment]
Lol, that's essentially what the Ruby CoC is for. Of course you're welcome to do as you please @medvednikov. 

At some point it's more than likely some community members will ask for something "more comprehensive" than that (as we've seen with some other projects) and I was hoping to catch that now with a simple one vs someone demanding that the project to adopt one of the more insane CoCs that create debate and problems for the community as well as ill will in and of itself.

**Comment by ylluminarious** [comment]
Yes, unfortunately someone may come along down the line and start a drama to adopt a much more strict and cumbersome code of conduct, which is what happened in the [Opal](https://github.com/opal/opal/issues/942) and [Ruby](https://bugs.ruby-lang.org/issues/12004) communities. The initial push in these communities was to adopt the [Contributor Covenant,](https://www.contributor-covenant.org/) which is much more strict and places a huge onus on project maintainers.

Fortunately, both Ruby and Opal ended up adopting a much less fanatical CoC (which is the one seen above), but with the drama that V has already been facing in Slack, HN, and elsewhere, it would have been nice to preempt more drama by having a simple, lax CoC already in place. Too many people nowadays think that you *need* to have a `CONDUCT.md` file in your repository and they're not going to take `Be excellent to each other` as an answer. I'm no fan of this push for codes of conduct either, but I felt like the Ruby CoC was a good compromise between common sense / simplicity and having a formally laid out CoC.

Just my 2¢.

**Comment by ylluminate** [comment]
Yes, @ylluminarious is 100% correct and at some point we'll probably find ourselves coming back to this topic. For now @medvednikov went ahead and accepted the very simple one that he mentioned above and we'll just have to cross that bridge when the time comes. 🤷🏼‍♂️

**Pull request closed without merging**